src/services/prisma-datastore/PrismaDataProvider.ts

```diff
@@ -0,0 +1,234 @@
+import Prisma, { PrismaClient, ReadingSource, ReadingSourceType } from "@prisma/client";
```

Comment:
For me locally TypeScript doesn’t like any of these imports. For example.

> Module '"@prisma/client"' has no exported member 'PrismaClient'.

Reply:
did you do `yarn install`? That should create what is needed.

Reply:
fyi the errors did go away after I restarted Visual Studio Code. Something to keep in mind if others hit this same problem.